src/types/block.rs

```diff
@@ -1,6 +1,49 @@
 use serde::{Serialize, Serializer};
 use types::{Bytes, U128, U256, H256, H160, H2048};
 
+/// The block type returned from RPC calls.
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct BlockHeader {
```

Comment:
Duplication here with the Block struct, added this "header" struct to avoid making breaking changes, possible alternatives would be to make the remaining fields `Option<...>`s or make the Block struct contain a BlockHeader which likely would need a custom Deserialize implementation and also would change the struct layout.